beacon-chain/blockchain/service.go

```diff
@@ -99,6 +99,13 @@ func (c *ChainService) Start() {
 	if beaconState != nil {
 		log.Info("Beacon chain data already exists, starting service")
 		c.genesisTime = time.Unix(int64(beaconState.GenesisTime), 0)
+		genesisState, err := c.beaconDB.GenesisState(c.ctx)
+		if err != nil {
+			log.Fatalf("Could not retrieve genesis state from db: %v", err)
+		}
+		if err := c.forkChoiceStore.GenesisStore(c.ctx, genesisState); err != nil {
+			log.Fatalf("Could not start fork choice service: %v", err)
+		}
```

Comment:
While you are here, can you also emit to the chain state initialized feed?

Reply:
example: https://github.com/prysmaticlabs/prysm/commit/78e4432468474037b92e7554d80d8b0c2542e64c